src/OpenTelemetry.Instrumentation.Dependencies/HttpClientInstrumentation.cs

```diff
@@ -42,7 +42,7 @@ public HttpClientInstrumentation(ActivitySourceAdapter activitySource)
         /// <param name="options">Configuration options for dependencies instrumentation.</param>
         public HttpClientInstrumentation(ActivitySourceAdapter activitySource, HttpClientInstrumentationOptions options)
         {
-            this.diagnosticSourceSubscriber = new DiagnosticSourceSubscriber(new HttpHandlerDiagnosticListener(options, activitySource), options.EventFilter);
+            this.diagnosticSourceSubscriber = new DiagnosticSourceSubscriber(new HttpHandlerDiagnosticListener(options, activitySource), (activitySource, arg1, arg2) => options?.EventFilter(activitySource, arg1) ?? true);
```

Comment:
can options be null ever?

Reply:
I just double-checked, no can't be null. All the instrumentation classes are internal, only way to create them is through the builder which news up the options instance.